src/nxdoc/nuxeo-add-ons/nuxeo-enhanced-viewer/nuxeo-enhanced-viewer-release-notes.md

```diff
@@ -0,0 +1,126 @@
+ - Sound can now be turned off when annotating a video
+ - Bug fixes (MP3 rendition, preview on HTML , email and ZIP, repeat mode,…)
+
+### Improvements
+
+#### Download with FDF annotations
+
+The "Download with FDF annotations" action allows to download a document as a PDF with annotations in the Forms Data Format (FDF). Then you can for instance open the document in Adobe Reader and see the annotations as if you have created them into Adobe area.
+
+See [NEV-323](https://jira.nuxeo.com/browse/NEV-323).
+
+#### Turn on/off sound when annotating a video
+
+It is now possible to turn off the sound when annotating a video.
+
+See [NEV-45](https://jira.nuxeo.com/browse/NEV-45).
+
+### UI/UX
+
+#### Replace sticky notes pointers by pointers with numbers
+
+The sticky notes annotations have a new look & feel. They now include a number into the pointer to easily link the annotation in the document to the text into the left section of the Annotations tab.
+
+See [NEV-349](https://jira.nuxeo.com/browse/NEV-349).
+
+#### Better icons into the Arrow annotation bar
+
+The arrow annotation bar includes a button to customise each side of the arrow. This new version improves those buttons icons for a better visibility.
+
+See [NEV-238](https://jira.nuxeo.com/browse/NEV-238).
+
+#### Empty annotation look & feel
+
+The left annotation section has been improved for empty annotations, in order to reflect that the annotation is editable
+
+See [NEV-250](https://jira.nuxeo.com/browse/NEV-250).
+
+### Performance and Stability
+
+The new version of ARender includes stability improvements, mostly on the renditions and conversions sides (cf. major fixes section).
+
+### Major Fixes
+
+#### Rendition of MP3 files
+
+MP3 files are now correctly rendered in the Annotations tab.
+
+See [NEV-332](https://jira.nuxeo.com/browse/NEV-332).
+
+#### Rendition of HTML files
+
+Some HTML files were not correctly rendered in the Annotations tab.
+
+See [NEV-354](https://jira.nuxeo.com/browse/NEV-354).
+
+### Renditions of emails
+
+The rendition of emails has been improved with better support of eml and msg files.
+
+We also fix language issue.
+
+See [NEV-370](https://jira.nuxeo.com/browse/NEV-370) and [NEV-330](https://jira.nuxeo.com/browse/NEV-330).
+
+### Renditions of ZIP files
+
+ZIP files are now correctly rendered in the Annotations tab.
+
+See [NEV-370](https://jira.nuxeo.com/browse/NEV-370).
+
+### Stop repeat mode button is not working well
+
+Clicking on the button with repeat mode didn't correctly stop the repeat mode, involving to create a last annotation before it stopped.
+
+See [NEV-335](https://jira.nuxeo.com/browse/NEV-335).
+
+### ARender health and liveness resources missing
+
+Following authentication changes, some resources where missing preventing to correctly display the ARender liveness and health pages.
+
+See [NEV-339](https://jira.nuxeo.com/browse/NEV-339) and [NEV-338](https://jira.nuxeo.com/browse/NEV-338)
+
```

Comment:
This has been shipped in th last 10.3, not sure it is interesting to mention it?